### PR TITLE
website/docs: prepare 2023.3 release notes

### PR DESCRIPTION
**Pull request by BeryJu**

### CODE REVIEW

File: website/docs/releases/2023/v2023.3.md [hunk 1]
```diff
@@ -0,0 +1,1307 @@
...
+-   sources/ldap: improve error handling for password complexity (#4780)
+-   sources/oauth: fix not all token errors being logged with response
+-   sources/plex: fix check_token error unusable if token is empty (#4834)
+-   stages/authenticator_sms: fix twilio sending (#4829)
+-   stages/user_login: add option to terminate other sessions (#4754)
+-   tests/e2e: use example blueprints for testing (#4805)
+-   web: fetch custom.css via fetch and add stylesheet (#4804)
+-   web: toggle dark/light theme manually (#4876)
+-   web/admin: fix chart display with no sources (#4782)
+-   web/admin: set wizard's valid correctly when opened and radio is already selected (#4821)
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.3.md:71]:
```suggestion
-   web/admin: fix issue with wizard's Next button incorrectly disabled when radio button is already selected (#4821)
```

File: website/docs/releases/2023/v2023.3.md [hunk 2]
```diff
@@ -0,0 +1,1307 @@
...
+-   web/admin: workaround for tenant certificate selection being cut off (#4820)
+-   web/elements: add loading spinner for charts, render middle text with css
```
Comment (tanberry) [inline website/docs/releases/2023/v2023.3.md:73]:
```suggestion
-   web/elements: add loading spinner for charts, render middle text with CSS
```

**Review by tanberry — COMMENTED** [review]
LGM!

File: website/docs/releases/2023/v2023.3.md [hunk 3]
```diff
@@ -0,0 +1,1307 @@
...
+    -   The custom.css file is now loaded in ShadowDOMs, allowing for much greater customization, as previously it was only possible to style elements outside of the ShadowDOM.
+    -   Previously, authentik would automatically switch between dark and light theme based on the users' browsers' settings. This can now be overridden to either force the light or dark theme, per user/group/tenant.
+
+## Upgrading
+
+This release does not introduce any new requirements.
+
+### docker-compose
+
+Download the docker-compose file for 2023.3 from [here](https://goauthentik.io/version/2023.3/docker-compose.yml). Afterwards, simply run `docker-compose up -d`.
```
[the remark this answers is not in the document]
Comment (BeryJu) [reply website/docs/releases/2023/v2023.3.md:27]: This block is always copied from the template, see https://github.com/goauthentik/authentik/blob/main/website/docs/releases/_template.md

File: website/docs/releases/2023/v2023.3.md [hunk 4]
```diff
@@ -0,0 +1,1307 @@
...
+
+Update your values to use the new images:
+
+```yaml
+image:
+    repository: ghcr.io/goauthentik/server
+    tag: 2023.3.0
+```
+
+## Minor changes/fixes
```
[the remark this answers is not in the document]
Comment (BeryJu) [reply website/docs/releases/2023/v2023.3.md:39]: All the changes listed below this point are commit messages/PR titles, just FYI

File: website/docs/releases/2023/v2023.3.md [hunk 5]
```diff
@@ -0,0 +1,1307 @@
...
+-   core: bootstrap email (#4788)
+-   core: enforce unique on names where it makes sense (#4866)
+-   core: Fix bug causing whitespace only names to raise exception when generating avatars (#4746)
+-   core: fix error when creating token without request in context
+-   core: Improve service account creation (#4751)
+-   events: fix m2m_change events not being logged
+-   flows: change default flow stage binding settings (#4784)
+-   flows: planner error handling (#4812)
+-   internal: fix crash when port 9000 is in use (#4863)
+-   providers: SCIM (#4835)
```
[the remark this answers is not in the document]
Comment (BeryJu) [reply website/docs/releases/2023/v2023.3.md:53]: no, but hence it's also listed on the New changes on the first header, I just like to include noteworthy changes in this list even if they are mentioned above

**Pull request merged**